src/components/Dashboard.tsx

```diff
@@ -37,8 +47,86 @@ export const Dashboard = () => {
       ...card,
     }))
 
+  const onDragEnd = ({ destination, source, draggableId, type }: any) => {
```

Comment:
It is easy to sneak peek from context:
![image](https://user-images.githubusercontent.com/23147514/154568540-f64d53f7-2306-446e-81c1-9b0300041ef4.png)

```suggestion
  import { DragDropContext, DropResult } from 'react-beautiful-dnd'
  
  const onDragEnd = ({ destination, source, draggableId, type }: DropResult) => {
```

Reply:
Also, this function is quite big and complex. I would suggest to, at least, split it to smaller functions and use them inside `onDragEnd`.